test/cli/rbi-final/r.rbi

```diff
@@ -0,0 +1,6 @@
+# typed: true
+class C
+  extend T::Sig
+  sig(:final) {void}
+  def foo; end
+end
```

Comment:
Can you rewrite this as a testdata test? You should be able to write multi-file testdata tests (even with RBI files) like this:

https://github.com/sorbet/sorbet/pull/1302/files#diff-9cbf0df41ae564f7ab8a51cac66422df

Reply:
if you want to test that the ordering doesn't matter i'd suggest two sets of testdata tests, but i'd say that's probably out of scope.